<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCUIEdit v0.0.0rc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>961</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="TCUIEdit::property_browser::Browser" name="mainView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTCUIEdit">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuTCUIEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAaa">
   <property name="text">
    <string>aaa</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TCUIEdit::property_browser::Browser</class>
   <extends>QTreeView</extends>
   <header>src/components/property_browser/Browser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
